src/main/java/com/faforever/api/player/PlayerRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.faforever.api.player;
+
+import com.faforever.api.data.domain.Player;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface PlayerRepository extends JpaRepository<Player, Integer> {
```

Comment:
`@Repository` (actually we could remove it from all classes, since it's inherited, but on the other hand it makes it explicit that this is a repository bean). Up to you, but keep it consistent.